Add mention of taxonomy tagging instructions (#158)
mongodb/docs-meta: source/style-guide/markup/directives/metadata.txt

meta
----

Use ``.. meta::`` to add |html| meta tags to a page. Adding meta tags

> aids in searches both internally and through the Internet. You can add

Reviewer: ```suggestion
aids in searches both internally and on external search engines. You can add
```
A suggestion! 'on the internet' would work as well

Reviewer 2: +1

Author: Thanks, I incorporated the content in a rewrite of this intro section since it wasn't explaining the concepts very well.

both keywords and descriptions. To learn more, see
:ref:`seo-guidelines`.

.. important::

   MongoDB Technical Writers should use the meta keyword standards defined in

>    the "Taxonomy Tagging Instructions" internal wiki page.

Reviewer 2: Can we link?

Reviewer 3: I'm not sure if we can link to the internal wiki since the meta site is available to the public. I might be mistaken though.

Author: @corryroot I avoided linking because the wiki is only accessible by MongoDB employees, and the style guide is public. Let me know if you think it should be linked or if you have any other solutions in mind.
